### PR TITLE
feat(coveralls): go-lint implementations and fixes. (WIP)

### PR DESCRIPTION
**Pull request by yasirfolio3**

### CODE REVIEW

**Comment by mikeng13** [comment]
Resolve conflicts please!

**Comment by yasirfolio3** [comment]
> Resolve conflicts please!

done.

File: optimizely/event/dispatcher.go [hunk 1]
```diff
@@ -7,19 +7,22 @@ import (
 	"net/http"
 )
 
+// Dispatcher dispatches events
 type Dispatcher interface {
 	DispatchEvent(event LogEvent, callback func(success bool))
 }
 
-type HttpEventDispatcher struct {
+// HTTPEventDispatcher represents HTTPEventDispatcher object
```
Comment (mikeng13) [inline optimizely/event/dispatcher.go]: `HTTPEventDispatcher is the HTTP implementation of the Dispatcher interface`

File: optimizely/event/events.go [hunk 2]
```diff
@@ -1,91 +1,102 @@
 package event
 
+// EventContext respresents event context of a UserEvent
```
Comment (mikeng13) [inline optimizely/event/events.go]: `EventContext holds project-related contextual information about a UserEvent`

File: optimizely/event/events.go [hunk 3]
```diff
@@ -1,91 +1,102 @@
...
 	EventContext EventContext
-	VisitorID string
-	Impression *ImpressionEvent
-	Conversion *ConversionEvent
+	VisitorID    string
+	Impression   *ImpressionEvent
+	Conversion   *ConversionEvent
 }
 
+// ImpressionEvent respresents a impression event
```
Comment (mikeng13) [inline optimizely/event/events.go]: nit: `... an impression event`

Comment (yasirfolio3) [reply optimizely/event/events.go]: done, please review now.

**Pull request merged**